lms/static/sass/shared/_modal.scss

```diff
@@ -62,6 +62,11 @@
     padding-bottom: ($baseline/2);
     position: relative;
 
+    p {
+      font-size: .9em;
+      line-height: 1.4;
```

Comment:
If this code is new as well, why not use the ``line-height()`` mixin we have in the LMS for this - https://github.com/edx/edx-platform/blob/master/lms%2Fstatic%2Fsass%2Fbase%2F_mixins.scss#L11 ?